std/root.ln

```diff
@@ -189,6 +525,7 @@ export fn max(x: Orderable, y: Orderable): Orderable {
 }
 
 // Boolean and bitwise functions
+// TODO: Should Result-wrapped integers be accepted by the bitwise functions?
```

Comment:
Maybe? In the same vein as the `abs` functions above, where it doesn't result-wrap if both inputs are "bare"?